pkg/monitoring/prometheusagent/configmap.go

```diff
@@ -48,7 +48,7 @@ func (pas PrometheusAgentService) buildRemoteWriteConfig(ctx context.Context,
 	// Compute the number of shards based on the number of series.
 	headSeries, err := querier.QueryTSDBHeadSeries(ctx, cluster.Name)
 	if err != nil {
-		logger.Error(err, "failed to query head series")
```

Comment:
I would keep this as an error though

Reply:
+1 this is an error

Reply:
I suggested in the ticket to log a warning and create an alert in case of `observability_operator_mimir_head_series_query_errors_total` > X

Reply:
But why Can you not log the error and not return the error?

Reply:
Oh yeah my bad!